lib/config/configItems.joi.js

```diff
@@ -112,6 +112,14 @@ const probeServerJoi = joi.object({
     port: joi.number().required(),
 });
 
+const probeServerPerSite = joi.array().items(
```

Comment:
there is not validation to 'reject' empty array: not sure if this is by design, or just an omission.

* can this happen in s3c, e.g. maybe in case there is no `site` (no crr configured) → must be supported
* if not, would be better to add the required `joi` config to validate the array is not empty

Reply:
I added the `joi` config to verify array is not empty, as if we have an array it shouldnt be empty

If CRR is not configured then probeserver is supposed to use the "normal" config which is `probeServerJoi`
as done here: 
```
probeServer: joi.alternatives().try(
            probeServerJoi,
            probeServerPerSite
```